R/pkg/R/SQLContext.R

```diff
@@ -462,14 +462,19 @@ read.df <- function(sqlContext, path = NULL, source = NULL, ...) {
     source <- callJMethod(sqlContext, "getConf", "spark.sql.sources.default",
                           "org.apache.spark.sql.parquet")
   }
-  sdf <- callJMethod(sqlContext, "load", source, options)
```

Comment:
Its actually harder to use the chaining API from SparkR -- We'll need to get a jobj for sqlContext.read and then set schema on it and then call load on it. Not sure its worth to make 3 RPCs from R to JVM ?